app/config/translations/text_de.yml

```diff
@@ -56,40 +56,40 @@ text:
     text: Mit der rokka image API verteilen Sie Ihre Bilder ganz einfach auf Ihre Web-Präsenzen. Noch einfacher wird die Bedienung von rokka durch unsere Libraries für die gängigen Sprachen.
 
   pricing:
-    title: 'Preise: Einfach und transparent.'
+    title: 'Preise: Fair und transparent.'
     paid:
-      title: Ein Angebot für alle.
-      text: Bei rokka bezahlen Sie nur das, was Sie effektiv brauchen. Die Kosten werden Ihnen quartalsweise in Rechnung gestellt und anhand der monatlichen Aufstellung haben Sie eine genaue Übersicht über Ihre Kosten.
+      title: Professionell einsteigen
+      text: Bei rokka bezahlen Sie nur das, was Sie effektiv brauchen. Wir unterscheiden dabei in Speicherplatz und Traffic. Die Kosten werden Ihnen quartalsweise in Rechnung gestellt.
```

Comment:
good idea! Sounds a little bit cumbersome but still I agree that it might be the best choice for non-technical people. How do we proceed now? Shall I adapt the changes or will you do it within the approval process?